### PR TITLE
Feat(Sleek): Styling improvements + fixes

### PR DESCRIPTION
**Pull request by tandpfun**
Hi all!

I've made some general improvements to the sleek theme. I made these only for me at first, but decided to pr them as they seem like things that should be added to the theme in general. I'm probably not going to be around to change stuff from what I did here, but feel free to pr changes if there's any mistakes/problems with my code (there probably are lol).

**What I've done:**
 - I formatted all the css with prettier to make stuff more consistent, readable, and efficient
 - I added transitions in a lot of places where they were missing, like song lists, buttons, progress/volume bars, and much more!
 - I know there was already confirmation dialog fixes, but I made it look even better
 - I added the song duration back to the progress bar to float above the progress bar on hover (with transitions and everything!)
 - I made the song art album cover thing more rounded like the playlist covers
 - I made the side bars have some sort of gradient, isn't really too noticeable though.
 - I made the scrollbars smaller on windows & removed when not necessary.

**I previewed these changes mainly on the deep theme (as that's the one I use), there may be some problems with other themes, though I'm not sure.**

Thank you for checking this out, let me know if you have any questions!

Photos:

(It's hard to show the transitions & some other new stuff listed through these screenshots, so I recommend you try it out :)

### CODE REVIEW

File: Sleek/user.css [hunk 1]
```diff
@@ -1,359 +1,507 @@
...
+[aria-label='Playing'] {
+  color: var(--spice-text);
 }
 
-[aria-label="Playing"] {
-    color: var(--spice-text);
+/* Round Cover Art */
+.cover-art-image,
+.cover-art {
+  border-radius: 10%;
```
Comment (harbassan) [inline Sleek/user.css:118]:
```suggestion
.main-coverSlotCollapsed-container .cover-art-image {
  border-radius: 10%;
```

File: Sleek/user.css [hunk 2]
```diff
@@ -1,359 +1,507 @@
...
+  margin-top: 20px;
 }
 
 /* Main Play Button Change */
 .main-playButton-PlayButton.main-playButton-primary {
-    color: var(--spice-main);
-    background-color: var(--spice-play-button);
-    box-shadow: var(--spice-shadow) 0 5px 4px 0px;
+  color: var(--spice-text);
+  background-color: var(--spice-button);
```
Comment (harbassan) [inline Sleek/user.css:303]:
```suggestion
  color: var(--spice-button-text);
  background-color: var(--spice-play-button);
```

**Review by harbassan — CHANGES_REQUESTED** [review]
This is all great! The only two things that I suggest changing are:

The color changes you have made to the dialog boxes and the main play/pause button, because that change is incompatible with a lot of the schemes which have the same color for both variables

The rounded song art, because although I agree with the change in the smaller view which it seems like you have tested with, when expanded it doesn't work well at all.

**Comment by morpheusthewhite** [comment]
You may also want to update the screenshots of the other color schemes after checking that they are correctly visualized

**Comment by tandpfun** [comment]
> You may also want to update the screenshots of the other color schemes after checking that they are correctly visualized

Yep, I'll try to do that when I have time.

**Comment by morpheusthewhite** [comment]
> Yep, I'll try to do that when I have time.

Then, if this looks fine for @harbassan, I will merge it after that

**Comment by tandpfun** [comment]
> You may also want to update the screenshots of the other color schemes after checking that they are correctly visualized

Looking at the different themes, my changes don't look different than the current screenshot, so I think the current ones are fine (most changes were to things on hover & stuff not really visible in the screenshots). I did find a mistake I made with the play button being hidden for playlists & fixed it.

**Pull request merged**